qbft/controller.go

```diff
@@ -48,14 +50,21 @@ type Controller struct {
 }
 
 func NewController(
-	identifier []byte,
+	identifier types.MessageID,
 	share *types.Share,
 	domain types.DomainType,
 	config IConfig,
 ) *Controller {
 	return &Controller{
-		Identifier:          identifier,
-		Height:              -1, // as we bump the height when starting the first instance
+		Identifier: identifier,
+		// TODO<olegshmuelov>: fastssz does not support int but only uint.
+		// this why the Height type was changed from int64 to uint64
+		// The initial value of the height was changed from -1 to math.MaxUint64
+		// when we bump the height at the first instance we get 0 (math.MaxUint64 + 1 = 0)
+		// if we want to keep the Height to be int64 there are 2 possible options:
+		// 1. create a messageSSZ with uint64 instead of Height and transform to message with int64
+		// 2. open a pr for fastssz with the implementation that supports int
+		Height:              math.MaxUint64, // as we bump the height when starting the first instance
```

Comment:
fastssz does not support int but only uint.
this why the Height type was changed from int64 to uint64
the initial value of the height was changed from -1 to math.MaxUint64
when we bump the height at the first instance we get 0 (math.MaxUint64 + 1 = 0)
if we want to keep the Height to be int64 there are 2 possible options:
  1. create a messageSSZ with uint64 instead of Height and transform to message with int64
  2. [bump controller height when instance decides not when starts](https://github.com/bloxapp/ssv-spec/issues/62)